### PR TITLE
Feat/tr2 40 add sbt release

### PR DESCRIPTION
**Pull request by danryc**

### CODE REVIEW

File: version.sbt
Comment (rucek) [inline version.sbt]: **question:** Should we bump the version here already? Or rather wait until this is merged and autimatically released by CI?

Comment (danryc) [reply version.sbt]: Good point! I have changed it to 0.1.0-SHAPSHOT and check:

```
Release version [0.1.0] :
Next version [0.1.1-SNAPSHOT]
```

So fix was commited!

**Review by rucek — COMMENTED** [review]
**suggestion:** BTW - we decided that we want to bump the minor version on release. Since IIRC the default strategy is `Next`, which would bump the patch version, can you please make sure that it's the minor version that gets bumped?

**Review by kciesielski — COMMENTED** [review]
LGTM, +1 to Jacek's comment on the version bump strategy.

**Review by kciesielski — APPROVED** [review]

**Pull request merged**
